atomic-git/f22_spc/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# based on Dockerfile by Brent Baude
+
+FROM openscap-daemon-f22
+
+RUN dnf -y update && dnf -y groupinstall "Development Tools" && dnf -y install make wget git && dnf clean all
+
+RUN dnf remove -y openscap-containers && \
```

Comment:
Not necessary, but safer. If "manual" installation don't override package files, we can use wrong version to test.